code/modules/admin/verbs/individual_logging.dm

```diff
@@ -46,22 +46,27 @@
 	var/log_source = M.logging;
 	if(source == LOGSRC_CLIENT && M.client) //if client doesn't exist just fall back to the mob log
 		log_source = M.client.player_details.logging //should exist, if it doesn't that's a bug, don't check for it not existing
-
+	var/list/concatenated_logs = list()
 	for(var/log_type in log_source)
 		var/nlog_type = text2num(log_type)
 		if(nlog_type & ntype)
-			var/list/reversed = log_source[log_type]
-			if(islist(reversed))
-				reversed = reverseRange(reversed.Copy())
-				for(var/entry in reversed)
-					dat += "<font size=2px><b>[entry]</b><br>[reversed[entry]]</font><br>"
-			dat += "<hr>"
+			var/list/all_the_entrys = log_source[log_type]
+			for(var/entry in all_the_entrys)
+				concatenated_logs += "<b>[entry]</b><br>[all_the_entrys[entry]]"
+	if(length(concatenated_logs))
```

Comment:
```suggestion
	if(LAZYLEN(concatenated_logs))
```

Reply:
Isn't this functionally the same unless the whole list is made lazy?

Reply:
I did not define it as a lazylist, it's an empty list `var/list/concatenated_logs = list()`, though I suppose I could...?
If I did it's not likely to ever happen that an admin views the logs of something that has a non-existent list of logs. Any mob that ever has a client (read: all players) has at least ownership logs, and any client-less mob that emotes or speaks would have at least those logs as well.